Logger.Tests/LogFactoryTests.cs

```diff
@@ -6,6 +6,31 @@ namespace Logger.Tests
     [TestClass]
     public class LogFactoryTests
     {
-        
+        [TestMethod]
+        public void CreateLogger_NullFilePath()
+        { 
+            // Arrange
+            LogFactory logFactory = new LogFactory();
```

Comment:
nit: You can leverage target type new expression in C# 9.
```suggestion
            LogFactory logFactory = new();
```